### ISSUE TITLE
how to continue training from resumed checkpoint? just set checkpoint and learning rate?

### ISSUE THREAD

**Issue by HiSultryMan** [OFA-Sys/OFA#81]

**Comment by logicwong** [comment]
@HiSultryMan If you want to preserve the previous training states (e.g., learning rate, optimizer states), then you just need to set the checkpoint. If you want to reset the learning rate, you also need to set `--reset-optimizer`.
